vars/detectDockerAgent.groovy

```diff
@@ -4,9 +4,18 @@ Map call(Map args = [:]) {
     def inputManifest = lib.jenkins.InputManifest.new(readYaml(file: manifest))
     dockerImage = inputManifest.ci?.image?.name ?: 'opensearchstaging/ci-runner:ci-runner-centos7-v1'
     dockerArgs = inputManifest.ci?.image?.args
+    // Using default javaVersion as jdk-17
+    String javaVersion = 'jdk-17'
```

Comment:
The dockerArgs is the java version. More details in this PR

Reply:
`dockerArgs` is not only for java version, its all args for the container it can be also as `-p 8080:8080`, so I'm getting the jdk match and extracting the version, else it will take the jdk17.
Thank you